python/lsst/daf/butler/script/queryCollections.py

```diff
@@ -47,7 +51,11 @@ def queryCollections(repo, collection_type, flatten_chains, include_chains):
         collection names.
     """
     butler = Butler(repo)
+    kwargs = {}
+    if glob:
```

Comment:
since globToRegex can take an empty list it might be clearer to write this as:

```python
filter = globToRegex(glob)
if filter:
    kwargs["expression"] = filter
```